languages/python/sqlalchemy-oso/requirements.txt

```diff
@@ -1,2 +1,3 @@
 oso~=0.8.2
 sqlalchemy~=1.3.20
+sqlalchemy-stubs
```

Comment:
Is this needed here?

Reply:
Instead of a development dependency?

Reply:
Yeah might work that way.

Reply:
A user would need to install that to use type checking though if this library ships with types.